ui/src/app/pages/login/account-creation/account-creation-survey.tsx

```diff
@@ -111,16 +109,20 @@ export class AccountCreationSurvey extends React.Component<AccountCreationSurvey
   }
 
   toggleDisability(value) {
```

Comment:
Nit: methods named "toggle" generally mean "switch from one state to another every time I'm called", while this method and the one below simply update a field to a given boolean value. A slightly clearer name may be "updateDisability" or "setDisability".

Reply:
I may actually just get rid of these and use the `updateDemographicAttribute` cause its really the exact same thing.